apps/mobile/src/libs/components/background-wrapper/libs/maps/size-to-width-height-map.ts

```diff
@@ -0,0 +1,7 @@
+const sizeToWidthHeightMap = {
+	lg: { height: 160, width: 160 },
+	md: { height: 100, width: 100 },
+	sm: { height: 86, width: 86 },
+};
```

Comment:
And you can remove borderRadius from styles.ts
```suggestion
const sizeToStyleMap = {
	lg: { height: 160, width: 160, borderRadius: 160 } satisfies ViewStyle,
	md: { height: 100, width: 100, borderRadius: 100 } satisfies ViewStyle,
	sm: { height: 86, width: 86, borderRadius: 86 } satisfies ViewStyle,
};
```